tests/managedmcg_controller_test.go

```diff
@@ -82,6 +86,26 @@ var _ = Describe("ManagedMCGReconciler Reconcile", func() {
 			},
 		}
 
+		consoleDepFake := appsv1.Deployment{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "mcg-ms-console",
+				Namespace: namespace,
+			},
+		}
+		consolePluginFake := consolev1alpha1.ConsolePlugin{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "mcg-ms-console",
+			},
+			// Spec: consolev1alpha1.ConsolePluginSpec{
```

Comment:
please remove commented code

Reply:
It looks like this was introduced here: https://github.com/red-hat-storage/mcg-osd-deployer/pull/44/files#diff-95527ea8bbfbfa57170f8b6d8cf862f19fa4037cc7ea51e88e003bb982e3febeR99-R107.

Removed.